handler/asset_handler.go

```
@@ -0,0 +1,40 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/vkhichar/asset-management/contract"
+	"github.com/vkhichar/asset-management/customerrors"
+	"github.com/vkhichar/asset-management/service"
+)
+
+func ListAssetHandler(asset service.AssetService) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		asset, err := asset.ListAssets(r.Context())
+		if err == customerrors.NoAssetsExist {
+			fmt.Println("handler: No assets exist")
+			w.WriteHeader(http.StatusNotFound)
+			responseBytes, _ := json.Marshal(contract.ErrorResponse{Error: "no asset found"})
```

don't ignore error using _